### ISSUE TITLE
List looks only for maya representation

### ISSUE THREAD

**Issue by BigRoy** [ynput/ayon-maya#165]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior:

Currently the [list looks logic](https://github.com/ynput/ayon-maya/blob/8d263bc2d651aff03555250af462baf069086e70/client/ayon_maya/api/lib.py#L1846-L1857) collects any "look" product - but with other integrations also supporting to write looks and potentially maya in the future also possibly writing looks (but for usd) this should be fixed to exclude those.

### Expected Behavior:

The looks listed for the Look Assigner should only list those with a `ma` representation (or maybe `mb` if it can write both). It'd never work for any other look representation currently.

So basically, filter to look products with `ma` representations only.

### Version

1.0.0

### What platform you are running on?

Windows

### Steps To Reproduce:

1. Publish look from houdini (generates `usd`)
2. Open Look assigner for the asset in maya (load e.g. its model)
3. It lists the houdini look, even though it is not compatible.

### Are there any labels you wish to add?

- [X] I have added the relevant labels to the bug report.
